### PR TITLE
Fixed clipping and positioning

### PR DESCRIPTION
**Pull request by rzmk**
https://user-images.githubusercontent.com/30333942/110259981-60b1cc00-7f78-11eb-9078-0f727d794194.mp4

- Fixed pop-up clipping under Hero slogan (#12)
- Changed pop-up background positioning to "fixed" since the current overflow when scrolling to the bottom shows the main page
- Re-aligned some elements within pop-up body

Side thought:
The carousel background (transparent) isn't the same as the pop-up and changes to different clubs even though the current club description is different.

### CODE REVIEW

**Review by sriRacha21 — APPROVED** [review]

**Comment by sriRacha21** [comment]
Waiting on review from @matchaTiff

**Review by matchaTiff — APPROVED** [review]

**Pull request merged**
